### PR TITLE
Don't persist report data

### PR DESCRIPTION
**Pull request by badone**
## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/insights/module.py [hunk 1]
```diff
@@ -29,6 +29,26 @@ def __init__(self, *args, **kwargs):
         # health history tracking
         self._pending_health = []
         self._health_slot = None
+        self._store = {}
+
+    # The following three functions, get_store, set_store, and get_store_prefix
+    # mask the functions defined in the parent to avoid storing large keys
+    # persistently to disk as that was proving problematic. Long term we may
+    # implement a different mechanism to make these persistent. When that day
+    # comes it should just be a matter of deleteing these three functions.
```
Comment (dmick) [inline src/pybind/mgr/insights/module.py:38]: deleting

Comment (badone) [reply src/pybind/mgr/insights/module.py:38]: Thanks Dan

**Comment by badone** [comment]
https://pulpito.ceph.com/bhubbard-2021-07-22_03:36:13-rados-wip-badone-testing-distro-basic-smithi/

**Review by jdurgin — APPROVED** [review]
seems like a good approach to me!

**Review by neha-ojha — APPROVED** [review]
looks fine to me

**Pull request merged**